contributed_definitions/nyaml/NXpiezoelectric_material.yaml

```diff
@@ -0,0 +1,93 @@
+category: base
+doc: |
+  Material description and properties of the piezoelectric scanner materials.
+  The material of the Piezoelectric scanner is composed of polycrystalline solids and 
+  attached at the head of the tip. The material shows mechanical deformation due to the applied electric field. 
+
+type: group
+NXpiezoelectric_material(NXobject):
+  # Online links:
+  #   1. ttps://www.ulprospector.com/knowledge/2689/pe-piezoelectric-materials/#:~:text=piezoelectric%20ceramics%2C%20such%20as%20lithium,as%20polyvinylidene%20fluoride%20(PVDF).
+  #   2. https://info.piezo.com/hubfs/Data-Sheets/piezo-material-properties-data-sheet-20201112.pdf
+  #   3. https://application.wiley-vch.de/books/sample/3527345124_c01.pdf
+
+  name(NX_CHAR):
+    doc: |
+      The name of the material of the piezo scanner such as Lead Zirconate Titanates (PZTs).
+  chemical_formula(NX_CHAR):
```

Comment:
Have a look at `NXsubstance` for a proper description of what such a material is made of.

Reply:
NXsubstance is used to describe the chemical description.